### PR TITLE
Dread - Fix room typo

### LINKED ISSUE #4745: [Dread] "Darion" transporter typo
In the list of transporters and transporter destinations, Dairon is misspelled as "Darion," in the transporter destination,

"Ferenia/East Transport to Darion"

causing the list of transporters to read like this,

"Dairon/East Transport to Ferenia/Elevator to Ferenia - East Transport to **Darion**":
"Ferenia/East Transport to **Darion**/Elevator to Dairon - East Transport to Ferenia":

### PR DESCRIPTION
**Pull request by dyceron**
Fixes #4745

### CODE REVIEW

**Review by henriquegemignani — APPROVED** [review]

**Review by duncathan — CHANGES_REQUESTED** [review]
I'm pretty sure this requires a migration somewhere

**Comment by henriquegemignani** [comment]
> I'm pretty sure this requires a migration somewhere

Ah, I forgot to check if it's a starting location.

**Comment by duncathan** [comment]
what about the dock nodes? won't renaming those break rdvgame imports for seeds with those dock weaknesses randomized without a migration?

**Comment by henriquegemignani** [comment]
> what about the dock nodes? won't renaming those break rdvgame imports for seeds with those dock weaknesses randomized without a migration?

door lock is a thing, oops. yes it does break things

**Comment by henriquegemignani** [comment]
Well, that's why the tests failed. There's a test rdvgame with door lock rando that I bet failed.

File: test/test_files/log_files/dread/dread_dread_multiworld.rdvgame [hunk 1]
```diff
@@ -477,7 +477,7 @@
                 "Cataris/Transport to Dairon/Elevator to Dairon - Transport to Cataris": "Dairon/Transport to Cataris",
                 "Cataris/Teleport to Dairon/Teleporter to Dairon - Teleport to Cataris": "Dairon/Teleport to Cataris",
                 "Dairon/Teleport to Artaria/Teleporter to Artaria - Teleport to Dairon": "Artaria/Teleport to Dairon",
-                "Dairon/East Transport to Ferenia/Elevator to Ferenia - East Transport to Darion": "Ferenia/East Transport to Darion",
+                "Dairon/East Transport to Ferenia/Elevator to Ferenia - East Transport to Dairon": "Ferenia/East Transport to Dairon",
```
Comment (henriquegemignani) [inline test/test_files/log_files/dread/dread_dread_multiworld.rdvgame:480]: This pr is **not** allowed to touch any rdvgame.

The tests must pass with the rdvgames being untouched, as they represent all the rdvgames our users have.

**Review by henriquegemignani — CHANGES_REQUESTED** [review]

**Comment by ThanatosGit** [comment]
- Reverted the test fixes
- merged main
- wrote a migration (as teleporters are not present currently, only door lock is relevant)

File: test/test_files/patcher_data/dread/crazy_settings.json [hunk 2]
```diff
@@ -3678,10 +3678,10 @@
         }
     ],
     "text_patches": {
-        "GUI_DIFSELECTOR_LABEL_DESCRIPTOR_EXPERT": "Words Hash ($$$$$)",
+        "GUI_DIFSELECTOR_LABEL_DESCRIPTOR_EASY": "Words Hash ($$$$$)",
         "GUI_DIFSELECTOR_LABEL_DESCRIPTOR_NORMAL": "Words Hash ($$$$$)",
         "GUI_DIFSELECTOR_LABEL_DESCRIPTOR_HARD_UNLOCKED": "Words Hash ($$$$$)",
-        "GUI_DIFSELECTOR_LABEL_DESCRIPTOR_EASY": "Words Hash ($$$$$)",
+        "GUI_DIFSELECTOR_LABEL_DESCRIPTOR_EXPERT": "Words Hash ($$$$$)",
```
Comment (duncathan) [inline test/test_files/patcher_data/dread/crazy_settings.json:3684]: every time lmao

**Review by duncathan — APPROVED** [review]

**Pull request merged**